plugins/content_distribution/providers/comcast_mrss/ComcastMrssDistributionPlugin.php

```diff
@@ -22,8 +22,11 @@ public static function dependsOn()
 			self::CONTENT_DSTRIBUTION_VERSION_MINOR,
 			self::CONTENT_DSTRIBUTION_VERSION_BUILD);
 			
-		$dependency = new KalturaDependency(ContentDistributionPlugin::getPluginName(), $contentDistributionVersion);
-		return array($dependency);
+		$dependencyDistribution = new KalturaDependency(ContentDistributionPlugin::getPluginName(), $contentDistributionVersion);
+		$dependencyCaptions = new KalturaDependency(CaptionPlugin::PLUGIN_NAME);
```

Comment:
Please create constants in current class, you can't use constants of other plugins before you know they're existing and enabled.